msteams-platform/m365-apps/loop-component.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Guidelines for building Loop components
+description: In this article, learn how to build a Adaptive card based Loop component.
+ms.author: mobajemu
+ms.date: 06/15/2023
+ms.topic: tutorial
+ms.custom: m365apps
+ms.localizationpriority: medium
+---
+
+# Guidelines for building Loop components 
+
+## Overview
+
+Loop component is an evolution of Fluid component - a way to collaborate with a team through loop whether it’s in chat, email, meeting, or Loop page. Loop component stays in sync no matter how many places the content lives across Microsoft 365 apps.
+
+## Getting started
```

Comment:
Add a note before the getting started section:

[!NOTE]  
Adaptive Card-based Loop components are available in public preview.